### ISSUE TITLE
[yggcookie] Exception (yggcookie): FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Cloudflare Error: Cloudflare has blocked this request. Probably your IP is banned for this site, check in your web browser.: FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Cloudflare Error: Cloudflare has blocked this request. Probably your IP is banned for this site, check in your web browser. (Test)

### ISSUE THREAD

**Issue by ErrGenesis** [FlareSolverr/FlareSolverr#246]
remove i patch this with screen and puppeteer kiss

**Comment by ngosang** [comment]
Fixed in FlareSolverr 2.1.0

**Comment by Alphachoups** [comment]
> Fixed in FlareSolverr 2.1.0

Not fixed in 2.2.2 or problem reappeared at least.

**Comment by ilike2burnthing** [comment]
If you're using v2.2.2 then you're almost a year behind. Current release is v2.2.10, and beta is v3.

**Comment by Alphachoups** [comment]
> If you're using v2.2.2 then you're almost a year behind. Current release is v2.2.10, and beta is v3.

And... you are totally true. I just have to shut up my self :)
I have updated to the version 2.2.10 and it's working now, so thank you very much.

Only excuse i can find it's because i'm not very familar with docker and when I create a new docker with the "Latest" Image version i'm waiting that the latest version will be deployed.. but no.. I had to choose the 2.2.10 manually to have it.
